scripts/jenkins/track-upstream-and-package.pl

```diff
@@ -250,6 +256,8 @@ ()
 {
   system("mkdir -p $ENV{HOME}/.obs/cache/tar_scm/{incoming,repo,repourl}");
   system(qq(echo '[tar_scm]\nCACHEDIRECTORY="$ENV{HOME}/.obs/cache/tar_scm"' > ~/.obs/tar_scm));
+} else {
+  $ENV{"CACHEDIRECTORY"} = "$ENV{HOME}/.obs/cache/tar_scm"
```

Comment:
can we combine this with line 258 somehow?? is the problem that the config file that is being created there is not actually there anymore on the server? maybe just remove the unless() ?

Reply:
You're right! The problematic line is 258. There's no need to add '[tar_scm]' header as it is accounted for already in here.

https://github.com/guangyee/obs-service-tar_scm/blob/master/TarSCM/config.py#L27
https://github.com/guangyee/obs-service-tar_scm/blob/master/TarSCM/config.py#L67-#L70

I'll remove that from line 258.